### PR TITLE
add OBS development builds

### PR DESCRIPTION
**Pull request by getchoo**
development builds for openSUSE 15.4 and Tumbleweed are now available on the [OBS](https://build.opensuse.org/project/show/home:getchoo). this should probably be recommended as the COPR development builds only support Tumbleweed and OBS repos are just a lot easier to use on SUSE - though the COPR Tumbleweed builds will still be available

### CODE REVIEW

**Comment by AshtakaOOf** [comment]
Add `&logoColor=white` at the end of the shields, I forgot to do that so they can be pretty hard to see.

File: src/wiki/development/development-builds.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ Portable dev-builds are also provided; for both Windows and Linux.
...
+[![prismlauncher-git](https://img.shields.io/badge/aur-prismlauncher--git-blue?style=flat-square&logo=archlinux)](https://aur.archlinux.org/packages/prismlauncher-qt5-git/) [![prismlauncher-git](https://img.shields.io/badge/aur-prismlauncher--qt5--git-blue?style=flat-square&logo=archlinux)](https://aur.archlinux.org/packages/prismlauncher-git/) [![prismlauncher-git](https://img.shields.io/badge/mpr-prismlauncher--git-orange?style=flat-square&logo=debian)](https://mpr.makedeb.org/packages/prismlauncher-git) [![prismlauncher-nightly](https://img.shields.io/badge/copr-prismlauncher--nightly-blue?style=flat-square&logo=fedora)](https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/) [![prismlauncher-nightly](https://img.shields.io/badge/OBS-prismlauncher--nightly-brightgreen?style=flat-square&logo=opensuse)](https://build.opensuse.org/project/show/home:getchoo) [![prismlauncher-9999](https://img.shields.io/badge/gentoo-prismlauncher--9999-purple?style=flat-square&logo=gentoo)](https://packages.gentoo.org/packages/games-action/prismlauncher)
```
Comment (AshtakaOOf) [inline src/wiki/development/development-builds.md:20]: This should look better:
`[prismlauncher-git](https://aur.archlinux.org/packages/prismlauncher-qt5-git/) [prismlauncher-git](https://aur.archlinux.org/packages/prismlauncher-git/) [prismlauncher-git](https://mpr.makedeb.org/packages/prismlauncher-git) [prismlauncher-nightly](https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/) [prismlauncher-nightly](https://build.opensuse.org/project/show/home:getchoo) [prismlauncher-9999](https://packages.gentoo.org/packages/games-action/prismlauncher)`

[prismlauncher-git](https://aur.archlinux.org/packages/prismlauncher-qt5-git/) [prismlauncher-git](https://aur.archlinux.org/packages/prismlauncher-git/) [prismlauncher-git](https://mpr.makedeb.org/packages/prismlauncher-git) [prismlauncher-nightly](https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/) [prismlauncher-nightly](https://build.opensuse.org/project/show/home:getchoo) [prismlauncher-9999](https://packages.gentoo.org/packages/games-action/prismlauncher)

File: src/download/linux.md [hunk 2]
```diff
@@ -235,8 +235,12 @@ zypper addrepo https://download.opensuse.org/repositories/home:getchoo/openSUSE_
 zypper refresh
 # stable releases (Qt6 version, only for Tumbleweed)
 zypper install prismlauncher
+# latest builds (Qt6 version, only for Tumbleweed)
+sudo dnf install prismlauncher-nightly
```
Comment (AshtakaOOf) [inline src/download/linux.md:239]: Opensuse use zypper not dnf from what I remember?

File: src/download/linux.md [hunk 3]
```diff
@@ -235,8 +235,12 @@ zypper addrepo https://download.opensuse.org/repositories/home:getchoo/openSUSE_
...
 # stable releases (Qt5 version, available for Leap and Tumbleweed)
 zypper install prismlauncher-qt5
+# latest builds (avalible for Leap and Tumbleweed)
+sudo dnf install prismlauncher-qt5-nightly
```
Comment (AshtakaOOf) [inline src/download/linux.md:243]: Same here

**Review by AshtakaOOf — CHANGES_REQUESTED** [review]

File: src/download/linux.md (hunk 2 above)
Comment (getchoo) [reply src/download/linux.md:239]: copy and paste fail (╯°□°）╯︵ ┻━┻

**Review by AshtakaOOf — APPROVED** [review]
:+1:

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**